### DISCUSSION TITLE
Paid Features

### DISCUSSION THREAD

**Discussion by manybotts** [yash-dk/TorToolkit-Telegram#124] [General]
@yash-dk I was wondering if you could add a custom caption for all files uploaded. If it's possible to add that feature am willing to pay?

**Comment by yash-dk** [comment]
> @yash-dk I was wondering if you could add a custom caption for all files uploaded. If it's possible to add that feature am willing to pay?

Well i am still working on v2.0.3 which will take another month or so to release. You can raise the issue i will add it then. If you want that asap dm me on telegram.
